packages/python-numpy/build.sh

```diff
@@ -31,7 +32,7 @@ termux_step_configure() {
 	. ${_CROSSENV_PREFIX}/bin/activate
 	build-pip install wheel
 
-	LDFLAGS+=" -lpython${_PYTHON_VERSION} -lm"
+	LDFLAGS+=" -lpython${_PYTHON_VERSION} -landroid-complex-math"
```

Comment:
`-lm` should still be necessary because `libandroid-complex-math` is not a superset of libm.